data/darktableconfig.xml.in

```diff
@@ -360,6 +360,27 @@
     <shortdescription>tags case sensitivity</shortdescription>
     <longdescription>tags case sensitivity. without the Sqlite ICU extension, insensivity works only for the 26 latin letters</longdescription>
   </dtconfig>
+  <dtconfig>
```

Comment:
see comment below.

Reply:
I'll check, but this lines seems clearly come from an old attempt on my side...

Reply:
Not sure to understand the comments here, but the information about sensitivity are correct (I remember having discussed with Pascal to find the right wording (latin or ascii letters).